Terminal.Gui/View/Adornment/Adornment.cs

```diff
@@ -141,7 +141,7 @@ public override void OnDrawContent (Rect contentArea)
             }
         }
 
-        TextFormatter?.Draw (screenBounds, normalAttr, normalAttr, Rect.Empty, false);
+        TextFormatter?.Draw (screenBounds, normalAttr, normalAttr, Rect.Empty, TextFormatter.FillRemaining);
```

Comment:
Why not just change the `false` to `true`?

Reply:
Because that way we are forcing to used it as true. By te way this parameter is true by default and in this case we don't need to pass it into the parameter. With a property we let the user the choice to use his value to pass into the parameter.